Adds updated action-per-second wording (#2639)
temporalio/documentation: docs-src/cloud/limits.md

**What is the limit of Actions per second?**

Each Namespace has a rate limit, which is measured in Actions per second (APS).

> A namespace may be throttled when its throughput becomes too high.

Reviewer: ```suggestion
A Namespace may be throttled when its throughput becomes too high.
```

> Throttling means limiting the rate at which actions are performed to prevent overloading the system.

Reviewer 2: ```suggestion
Throttling means limiting the rate at which Actions are performed to prevent the Namespace from exceeding its APS limit.
```

don't want to give the impression that going over 200 APS would overload the system. also, imagine it would take all namespaces in a cluster going significantly over their limits at the same time to actually impact the cluster performance

A Namespace's default limit is set at 200 APS and automatically adjusts based on recent usage (over the prior 7 days).

> Your throughput limit will never fall below this default value.

Reviewer: The 200 APS value or the value that it automatically adjusts to?

For example, if it adjust upward to 300 APS, will the system ever decrease that?
Or will it never decrease below 200?

Author: Latest info is "Regarding automatic adjustment, it’s not correct to say it’s adjusting down - as we’ll never adjust to lower than default (unless customer requests it) and automatic adjustment will only bump the limit higher than default based on usage." My understanding is that it can revert slowly back down but never go below that default.

Author: Feedback from earlier iteration that did not make it in yet: https://github.com/temporalio/documentation/pull/2639

> Your throughput limit will never fall below this default value.

Reviewer 2: ```suggestion
Your APS limit will never fall below this default value.
```

"APS Limit" is the title used in the UI—suggest using it throughout instead of "throughput limit"
